proto/metadata.proto

```diff
@@ -50,6 +50,12 @@ message Metadata {
     // Device does not support interface/ipv4/enabled,
     // so suppress configuring this leaf.
     bool ipv4_missing_enabled = 1;
+    // Device does not support fragmentation bit for traceroute.
+    bool traceroute_fragmentation = 2;
+    // Device only support UDP as l4 protocol for traceroute.
+    // Use this field to set default l4 protocol as UDP and skip the tests that
```

Comment:
How about we delete this line and the next. I think simple, declarative specifications are easier to maintain, and I don't think we should commit ourselves to consistently documenting what is skipped by enabling a deviation.

Reply:
done